src/components/molecules/ScheduleItemNG/ScheduleItemNG.tsx

```diff
@@ -40,12 +40,18 @@ export interface ScheduleItemNGProps {
 }
 
 export const ScheduleItemNG: React.FC<ScheduleItemNGProps> = ({ event }) => {
-  const { currentVenue: eventVenue } = useRelatedVenues({
+  const { currentVenue: eventVenue, relatedVenues } = useRelatedVenues({
     currentVenueId: event.venueId,
   });
+
+  const relatedVenuesRooms = relatedVenues
+    ?.map((venue) => venue.rooms ?? [])
+    .flat()
```

Comment:
i guess you can use `flatMap` here instead of `map` + `flat`

Reply:
Nice catch.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2064/commits/0b02813c33eada2fe99b1a1e3c16dd15dda5955d